consensus/safety-rules/src/tests/suite.rs

```diff
@@ -56,7 +56,7 @@ pub type Callback = Box<
     ),
 >;
 
-pub fn run_test_suite(safety_rules: &Callback) {
+pub fn run_test_suite(safety_rules: &Callback, decoupled_execution: bool) {
```

Comment:
hmm, does test_bad_execution_output work with decoupled_execution = true?

Reply:
I tried it locally and I didn't see a failure..?